ECIPs/ecip-1056.md

```diff
@@ -68,13 +78,21 @@ increases its functionality and should be considered a feature upgrade rather th
 
 Adoption of the content of this ECIP requires a hard fork as it introduces changes that are not backward compatible.
 
-The following clients with Ethereum Classic support implement the _Constaninople_ and _Petersburg_ features currently:
+The following clients with Ethereum Classic support __implement the _Constaninople_ and _Petersburg_ opcodes__ currently:
 
 - Geth Classic: no support
 - Parity Ethereum: all features due to Ethereum Foundation compatibility
 - Multi Geth: all features due to Ethereum Foundation compatibility
 - Mantis: all features due to Ethereum Foundation compatibility
 
+The following clients with Ethereum Classic support __implement the EIP1702-derived Generalized Account Versioning Scheme__:
+
+- Geth Classic: no support
+- Parity Ethereum: no support (?)
+- Multi Geth: no support
+- Mantis: no support (?)
```

Comment:
Mantis does not support Ethereum Classic anymore, so no need to consider it here

Reply:
Done.